satpy/readers/sar_c_safe.py

```diff
@@ -636,10 +636,15 @@ def _get_lonlatalts_uncached(self):
 
         fine_points = [np.arange(size) for size in shape]
         x, y, z = lonlat2xyz(gcp_lons, gcp_lats)
-        interpolator = MultipleGridInterpolator((ypoints, xpoints), x, y, z, gcp_alts)
-        hx, hy, hz, altitudes = interpolator.interpolate(fine_points, method="cubic", chunks=self.chunks)
-        longitudes, latitudes = xyz2lonlat(hx, hy, hz)
 
+        kx = 2
+        ky = 2
```

Comment:
I think these can be removed and given directly as numbers, their meaning is clear as they are kwargs.